doc/rule-descriptions.md

```diff
@@ -129,11 +128,12 @@ Rules that do not necessarily conform to WCAG success criterion but are industry
-| [color-contrast-enhanced](https://dequeuniversity.com/rules/axe/4.9/color-contrast-enhanced?application=RuleDescription)           | Ensures the contrast between foreground and background colors meets WCAG 2 AAA enhanced contrast ratio thresholds | Serious | cat.color, wcag2aaa, wcag146, ACT              | failure, needs&nbsp;review | [09o5cg](https://act-rules.github.io/rules/09o5cg) |
-| [identical-links-same-purpose](https://dequeuniversity.com/rules/axe/4.9/identical-links-same-purpose?application=RuleDescription) | Ensure that links with the same accessible name serve a similar purpose                                           | Minor   | cat.semantics, wcag2aaa, wcag249               | needs&nbsp;review          | [b20e66](https://act-rules.github.io/rules/b20e66) |
-| [meta-refresh-no-exceptions](https://dequeuniversity.com/rules/axe/4.9/meta-refresh-no-exceptions?application=RuleDescription)     | Ensures &lt;meta http-equiv=&quot;refresh&quot;&gt; is not used for delayed refresh                               | Minor   | cat.time-and-media, wcag2aaa, wcag224, wcag325 | failure                    | [bisz58](https://act-rules.github.io/rules/bisz58) |
+| Rule ID                                                                                                                            | Description                                                                                                       | Impact   | Tags                                                                     | Issue Type                 | ACT Rules                                          |
+| :--------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------- | :------- | :----------------------------------------------------------------------- | :------------------------- | :------------------------------------------------- |
+| [color-contrast-enhanced](https://dequeuniversity.com/rules/axe/4.9/color-contrast-enhanced?application=RuleDescription)           | Ensures the contrast between foreground and background colors meets WCAG 2 AAA enhanced contrast ratio thresholds | Serious  | cat.color, wcag2aaa, wcag146, ACT                                        | failure, needs&nbsp;review | [09o5cg](https://act-rules.github.io/rules/09o5cg) |
+| [heading-order](https://dequeuniversity.com/rules/axe/4.9/heading-order?application=RuleDescription)                               | Ensures the order of headings is semantically correct                                                             | Moderate | cat.semantics, wcag2aaa, wcag2410, a11y-engine, a11y-engine-experimental | failure, needs&nbsp;review |                                                    |
```

Comment:
what is the usage ?

Reply:
Generated on commit. It is a text file with details about the rule. Since heading is moved from best-practices so the description changed